package.json

```diff
@@ -43,7 +43,7 @@
     "@ember/optional-features": "^1.3.0",
     "@glimmer/component": "^1.0.0",
     "@tracerbench/core": "4.0.10",
-    "@types/ember": "^3.1.1",
+    "@types/ember": "^3.16.2",
```

Comment:
This is a significant change. If we don't have any negative test cases for type information in this repo, I would spot check a few things to ensure that there aren't any regressions that inappropriately widen types (i.e., something becomes an `any` when it should be something more specific -- obviously a compiler will not object to this, but it's still a problem)